internal/proto/workflow/v2/workflow.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package proto.workflow.v2;
+
+option go_package = "github.com/tinkerbell/tink/internal/proto/workflow/v2;workflow";
+
+service WorkflowService {
+  // Workflows configures a bidirectional stream for a client to receive new workflows to run
+  // and submit workflow events.
+  rpc Workflows(stream WorkflowEvent) returns (stream Workflow) {}
+}
+
+
+message Workflow {
```

Comment:
Users may try to delete workflows mid-execution. How does the server communicate 'stop'?